### DISCUSSION TITLE
I justed created a new repository but my .ipynb is an "Invalid notebook".

### DISCUSSION THREAD

**Discussion by OscarErnst** [community/community#54277] [New to GitHub]
Hey GitHub Community,
I just made a new repository however I have trouble after i push my .jpynb file using VScode, anyone know what the problem could be?

**Comment by queenofcorgis** [comment]
Hi @OscarErnst  we're happy you're here! You are more likely to get a useful response if you are explicit about what your question entails, giving a few more details (like code) might help someone give you a nudge in the right direction.
